config-linux.md

```diff
@@ -545,6 +545,16 @@ Its value is either slave, private, or shared.
     ]
 ```
 
+## Mount Label
+
+`mountLabel` will set the Selinux context for the mounts in the container.
```

Comment:
Do we want cross-links between this and config.md's **`process.selinuxLabel`**?  I expect folks interested in one would also be interested in the other.